backend/app/services/base.py

```
@@ -0,0 +1,49 @@
+ModelType = TypeVar("ModelType")
+CreateSchemaType = TypeVar("CreateSchemaType")
+UpdateSchemaType = TypeVar("UpdateSchemaType")
+RepositoryType = TypeVar("RepositoryType", bound=BaseRepository)
+
+
+class BaseService(
+    Generic[ModelType, CreateSchemaType, UpdateSchemaType, RepositoryType]
+):
+    """Base service class for business logic operations."""
+
+    def __init__(self, repository: RepositoryType):
```

Ruff (0.12.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)